srcs/main.cpp

```diff
@@ -0,0 +1,40 @@
+#include "../base_headers/Socket.hpp"
+#include <fstream>
+#include <sstream>
+#include <string.h> // memset forbidden
+
+
+int main() {
+	// Create a socket (IPv4, TCP)
+	Socket socket(100, INADDR_ANY); // Listen 작업까지 되어 있는 소켓
+	size_t connections;
+
+	connections = socket.Accept(connections);
+
+	/* 테스트 진행 */
+		// Read from the connection
```

Comment:
```suggestion
		// STUB Read from the connection
```
VSCODE extension 중에 comment anchors 라고 있는데 그거 용으로 앵커 하나 달아둘게